src/components/Frame/components/Toast/tests/Toast.test.tsx

```diff
@@ -113,18 +113,22 @@ describe('<Toast />', () => {
 
     it('is called when the escape key is pressed', () => {
       const listenerMap: HandlerMap = {};
-      document.addEventListener = jest.fn((event, cb) => {
-        listenerMap[event] = cb;
-      });
+      jest
```

Comment:
Should we clear this in an afterEach?

Reply:
All these changes to use jest.spyOn are automatic fixes rather than manual ones. The end result is the same as the current behaviour. the symmetry of using jest.spyOn then mockRestore is nice but if i worked before it'll work after. I'd like to not get too bogged down in tidying up our tests as they work in the same way as before :)

I've updated the cases that you raised and found an extra one in Tabs.